src/v/v8_engine/executor.cc

```diff
@@ -152,8 +153,55 @@ ss::future<>
 executor_service::start(ss::alien::instance& instance, int64_t size) {
     co_await _executor.start_single(
       std::ref(instance), _core_for_executor_thread, size);
+    co_await _code_database.start_single();
```

Comment:
`start_single` always use core 0, are we ok with this limitation ?

Reply:
Yeah i think the idea here is that Vadim is going with a design that has the `_code_database` instance on 1 shard, instead of it being replicated across shards. Primarily this is done to save memory, only negative is that other shards will have to query shard 0 for this info but if querying this cache is off of the hot path i think thats fine.